client/src/pages/course/schedule.tsx

```diff
@@ -123,11 +123,7 @@ export function SchedulePage(props: CoursePageProps) {
           </Tooltip>
         </Col>
         <Col>
-          <UserSettings
-            tags={tags}
-            setStoredTagColors={setStoredTagColors}
-            storedTagColors={storedTagColors || {}}
-          />
+          <UserSettings tags={tags} setStoredTagColors={setStoredTagColors} storedTagColors={storedTagColors || {}} />
```

Comment:
I know it's not your change, but can we use an empty object as a default prop in `UserSettings` instead of defining it here all the time?

Reply:
fix